ormconfig.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name": "default",
+        "type": "sqlite",
+        "database": ":memory:",
+        "entities": ["./core/src/entities/*.entity.ts"],
+        "migrations": ["core/src/migrations/sqlite/*-*.ts"],
+        "migrationsRun": true,
+        "migrationsTableName": "migrations",
+        "cli": {
+            "migrationsDir": "core/src/migrations/sqlite"
+        }
```

Comment:
☝️ this database is purely for CI testing purposes, to make sure the sqlite migrations succeed when run from scratch